DOCS/man/osc.rst

```diff
@@ -21,7 +21,7 @@ The Interface
 ::
 
     +---------+----------+------------------------------------------+----------+
-    | pl prev | pl next  |  title                                   |    cache |
+    | pl prev | menu | pl next | title                                   cache |
```

Comment:
This menu is no longer for showing playlist only so probably better to put it on the left. `menu | pl prev | pl next`. What do you think?

Reply:
Well yeah that's where I put it initially and you asked to move it to the center.

Reply:
because I though this button would be to open playlist view, same as usoc.